src/librustc/ty/mod.rs

```diff
@@ -1363,6 +1364,7 @@ pub struct ReprOptions {
     pub packed: bool,
     pub simd: bool,
     pub int: Option<attr::IntType>,
+    pub align: Option<u64>,
```

Comment:
I'd use an `u16` and default it to `0` (we don't support larger alignments anyway - if you want to, you can use a pair of bytes  `Align` instead of just a byte, bringing the max alignment from `2^16` to `2^256`).